CRM/Badge/BAO/Layout.php

```diff
@@ -92,7 +92,7 @@ public static function create(&$params) {
     $params['is_default'] = CRM_Utils_Array::value('is_default', $params, FALSE);
     $params['is_reserved'] = CRM_Utils_Array::value('is_reserved', $params, FALSE);
 
-    $params['label_type_id'] = CRM_Core_OptionGroup::getValue('label_type', 'Event Badge', 'name');
+    $params['label_type_id'] = CRM_Core_PseudoConstant::getKey('CRM_Core_DAO_PrintLabel', 'label_type_id', 'Event Badge');
```

Comment:
Cannot be changed to ```CRM_Core_DAO_PrintLabel``` as the class don't exist

Reply:
OK - I have tested this line through the civicrm/admin/badgelayout?action=add&reset=1 screen